src/wishlist/wishlist.html

```diff
@@ -0,0 +1,124 @@
+                        <span class="card-price-initial">₹5000</span>
+                        <span class="card-discount block">50% OFF</span>
+                        <span class="card-desc block">Hey This Is A Tensiosn Free Jacket While Wearing This You Will Go All Tension Free, So Buy Now!</span>
+                        <div class="card-btn-box flex-center">
+                            <button type="button" class="btn btn-primary block">Add To Cart</button>
+                            <button type="button" class="btn btn-secondary-outline block">Buy Now</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="card-box-vertical">
+                    <div class="card-img flex-center">
+                        <img class="card-badge-top-right" src="../../assets/red-heart.svg">
+                        <img src="../../assets/jacket.svg" class="img-responsive">
+                    </div>
+                    <div class="card-text">
+                        <h3 class="card-title">Tension Free Jacket</h3>
+                        <span class="card-price-final">₹2500</span>
+                        <span class="card-price-initial">₹5000</span>
+                        <span class="card-discount block">50% OFF</span>
+                        <span class="card-desc block">Hey This Is A Tensiosn Free Jacket While Wearing This You Will Go All Tension Free, So Buy Now!</span>
+                        <div class="card-btn-box flex-center">
+                            <button type="button" class="btn btn-primary block">Add To Cart</button>
+                            <button type="button" class="btn btn-secondary-outline block">Buy Now</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="card-box-vertical">
+                    <div class="card-img flex-center">
+                        <img class="card-badge-top-right" src="../../assets/red-heart.svg">
+                        <img src="../../assets/jacket.svg" class="img-responsive">
+                    </div>
+                    <div class="card-text">
+                        <h3 class="card-title">Tension Free Jacket</h3>
+                        <span class="card-price-final">₹2500</span>
+                        <span class="card-price-initial">₹5000</span>
+                        <span class="card-discount block">50% OFF</span>
+                        <span class="card-desc block">Hey This Is A Tensiosn Free Jacket While Wearing This You Will Go All Tension Free, So Buy Now!</span>
+                        <div class="card-btn-box flex-center">
+                            <button type="button" class="btn btn-primary block">Add To Cart</button>
+                            <button type="button" class="btn btn-secondary-outline block">Buy Now</button>
+                        </div>
+                    </div>
```

Comment:
Try formatting with prettier for better indentation. Alt+shift+F on windows, Ctrl/Cmd+Shift+I for Mac. 
Logic looks clean and readable.

Reply:
thanks for the short cut will try it out,  currently, my prettier was having some issues don't know why